### PR TITLE
Adding main plant resource files

### PR DESCRIPTION
**Pull request by em-redstone**
Below are the changes/additions to the resource files for the main plant.

Will update as required,

Many thanks.

### CODE REVIEW

**Comment by charbull** [comment]
Hi please take a look at the checks there are error to fix at the ontology type validator: https://github.com/google/digitalbuildings/pull/122/checks?check_run_id=1113523209

Thank you

File: ontology/yaml/resources/HVAC/entity_types/PMP.yaml [hunk 1]
```diff
@@ -1,66 +1,77 @@
-# Copyright 2020 Google LLC
```
Comment (charbull) [inline ontology/yaml/resources/HVAC/entity_types/PMP.yaml:1]: can you please make sure you append instead of deleting and adding?
thx

**Comment by charbull** [comment]
can you please check some files are overwritten instead of being appended.
thx

**Comment by em-redstone** [comment]
Fixed line ending issue by applying "git config --global core.autocrlf true" to repository.

**Comment by tasodorff** [comment]
Source project has ended and this PR contains many things that would need to be changed. Recommend closing and creating a new PR in future, if needed.

**Pull request closed without merging**